templatestore/frontend/package-lock.json

```diff
@@ -10,12 +10,15 @@
       "license": "ISC",
       "dependencies": {
         "axios": "^0.19.2",
+        "bootstrap": "^5.1.3",
```

Comment:
Seems like we can use existing stylings/libraries instead of adding new ones. Remove these new additions.